loader/normalize.go

```diff
@@ -84,17 +84,22 @@ func normalize(project *types.Project, resolvePaths bool) error {
 		}
 		s.Environment = s.Environment.Resolve(fn)
 
-		err := relocateLogDriver(s)
+		err := relocateLogDriver(&s)
 		if err != nil {
 			return err
 		}
 
-		err = relocateLogOpt(s)
+		err = relocateLogOpt(&s)
 		if err != nil {
 			return err
 		}
 
-		err = relocateDockerfile(s)
+		err = relocateDockerfile(&s)
+		if err != nil {
+			return err
+		}
+
+		err = relocateScale(&s)
 		if err != nil {
```

Comment:
nit:
```suggestion
		if err := relocateScale(&s); err != nil {
```

Reply:
I hate this syntax, breaks readability ont he test condition imho

Reply:
However it saves on the few occasions when doing something like
```
err := something()
// forgetting to check err
if err := somethingElse(); err != nil {
}
```
This will make the compiler complain of an unused variable, whereas this doesn’t
```
err := something()
// forgetting to check err
err = somethingElse()
if err != nil 
{
}
```

But that’s fine, not a big deal.